pkgs/applications/audio/spotifyd/default.nix

```diff
@@ -30,6 +30,8 @@ rustPlatform.buildRustPackage rec {
     ++ stdenv.lib.optional withPulseAudio libpulseaudio
     ++ stdenv.lib.optional withPortAudio portaudio;
 
+  doCheck = false;
```

Comment:
Why this change?

Reply:
the checkPhase fails. I'm not sure why:
```
   ...
   Compiling spotifyd v0.2.14 (/private/tmp/nix-build-spotifyd-0.2.14.drv-0/source)
    Finished release [optimized] target(s) in 5m 20s
running tests
Running cargo test
error: the listed checksum of `/private/tmp/nix-build-spotifyd-0.2.14.drv-0/spotifyd-0.2.14-vendor/librespot-protocol/src/spirc.rs` has changed:
expected: 69d90e996fe6daf92580baa8bfe8a80b538c842b01505ec885e2e695c0460525
actual:   ed984f674231f831745048215c880f52929b004b65fc0bafe649d6516e701603

directory sources are not intended to be edited, if modifications are required then it is recommended that [replace] is used with a forked copy of the source
builder for '/nix/store/iy179xkb5wivnfh6mwfdqhnvsyd8z6nw-spotifyd-0.2.14.drv' failed with exit code 101
error: build of '/nix/store/iy179xkb5wivnfh6mwfdqhnvsyd8z6nw-spotifyd-0.2.14.drv' failed
```

Reply:
Neither am I. A quick grep shows that nixpkgs is full of `doCheck = false` so I think this solution is OK.